src/win32_window.c

```diff
@@ -2375,13 +2401,32 @@ const char* _glfwGetClipboardStringWin32(void)
 
 void _glfwSetThemeWin32(_GLFWwindow* window, _GLFWtheme* theme)
 {
-    _glfwInputError(GLFW_FEATURE_UNIMPLEMENTED, NULL);
+    if (!theme || theme->variation == GLFW_THEME_DEFAULT)
+    {
+        applySystemTheme(window->win32.handle);
+        return;
+    }
+
+    GLFWbool darkMode = theme->variation == GLFW_THEME_DARK;
+
+    DwmSetWindowAttribute(window->win32.handle,
+                          DWMWA_USE_IMMERSIVE_DARK_MODE,
+                          &darkMode,
+                          sizeof(darkMode));
 }
 
 _GLFWtheme* _glfwGetThemeWin32(_GLFWwindow* window)
 {
-    _glfwInputError(GLFW_FEATURE_UNIMPLEMENTED, NULL);
-    return NULL; // TODO: implement
+    _GLFWtheme* theme = &window->theme;
+
+    theme->variation = GLFW_THEME_LIGHT;
+    theme->flags = 0;
+
+    if (_glfw.win32.uxtheme.uxThemeAvailable && _glfw.win32.uxtheme.darkTitleAvailable) {
+        theme->variation = GLFW_THEME_DARK;
+    }
+
+    return theme;
 }
```

Comment:
Is it possible to acquire the theme colour set by the user, as well, and copy that into the colour member of the GLFWtheme?

Reply:
Yes, it's possible.

Reply:
Then that's the intended behaviour.

Reply:
The theme's GLFW_THEME_COLOR_MAIN flag must also be enabled.

Reply:
Thanks. :) `getAccentColor` should return a boolean for the availability of the colour. So, `GLFW_TRUE` if available, else `GLFW_FALSE`. Then if it returns true, the theme´s `GLFW_THEME_COLOR_MAIN` flag is set. I'll fix that myself after I merge dev-theming into theming, then resolve the conflicts with this PR.